src/tools/namespace_tool_core.cpp

```diff
@@ -125,30 +125,52 @@ int NameSpaceToolCore::GetFileSegments(const std::string& fileName,
     return 0;
 }
 
-int NameSpaceToolCore::CleanRecycleBin(const std::string& dirName) {
+int NameSpaceToolCore::CleanRecycleBin(const std::string& dirName,
+                                       const uint64_t expireTime) {
     std::vector<FileInfo> files;
     int res = client_->ListDir(curve::mds::RECYCLEBINDIR, &files);
     if (res != 0) {
         std::cout << "List RecycleBin fail!" << std::endl;
         return -1;
     }
+
+    auto needDelete = [](const FileInfo &fileInfo,
+                         uint64_t now,
+                         uint64_t expireTime) -> bool {
+        auto filename = fileInfo.filename();
+        std::vector<std::string> items;
+        ::curve::common::SplitString(filename, "-", &items);
+
+        uint64_t dtime;
+        auto n = items.size();
+        auto id = std::to_string(fileInfo.id());
+        if (n >= 2 && items[n - 2] == id
+            && ::curve::common::StringToUll(items[n - 1], &dtime)
+            && now - dtime < expireTime) {
+            LOG(INFO) << "Delete unexpected file: " << filename;
```

Comment:
should not log here, if lots of file in recyclebin, will log more redundant info

Reply:
> should not log here, if lots of file in recyclebin, will log more redundant info

done